fix(bitbucket): Update `slug` to follow owner_name/repo_name format

ghengeveld opened the pull request:
Fixes #136

ghengeveld commented:
@pvdlg Any chance this can be merged and released?

pvdlg commented:
Can you fix the tests?
See https://travis-ci.com/github/pvdlg/env-ci/jobs/489074757

ghengeveld commented on services/bitbucket.js:
```diff
@@ -13,7 +13,7 @@ module.exports = {
       buildUrl: `https://bitbucket.org/${env.BITBUCKET_REPO_SLUG}/addon/pipelines/home#!/results/${env.BITBUCKET_BUILD_NUMBER}`,
       branch: env.BITBUCKET_BRANCH,
-      slug: env.BITBUCKET_REPO_SLUG,
+      slug: env.BITBUCKET_REPO_FULL_NAME,
```
I missed the `buildUrl`. Clearly that should use `BITBUCKET_REPO_FULL_NAME` as well, as that is explicitly [defined](https://support.atlassian.com/bitbucket-cloud/docs/variables-and-secrets/) as `everything that comes after http://bitbucket.org/`

I think the issue exists but nobody actually encountered it or they worked around it. So did we.

ghengeveld commented:
Hey @gr2m @pvdlg I totally forgot about this PR for years. I've fixed the tests now and also updated `buildUrl` as was pointed out as something I overlooked.

gr2m reviewed (approved):
Thanks for finishing it 💐
